modules/addons/gofasnfeio/xml.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Módulo Nota Fiscal NFE.io para WHMCS
+ * @author		Original Author Mauricio Gofas | gofas.net
+ * @author		Updated by Link Nacional
+ * @see			https://github.com/nfe/whmcs-addon/
+ * @copyright	2020 https://github.com/nfe/whmcs-addon/
+ * @license		https://gofas.net?p=9340
+ * @support		https://github.com/nfe/whmcs-addon/issues
+ * @version		1.2.4
+ */
+
+//ini_set('display_errors', 1);
+//ini_set('display_startup_errors', 1);
+//error_reporting(E_ALL);
+
+require_once __DIR__ . '/../../../init.php';
+
+use WHMCS\Database\Capsule;
+
+$invoice_id = $_GET['invoice_id'];
+
+//echo $invoice_id;
```

Comment:
@souzadavi podemos remover esse comentários? se for necessário esse echo para debug, existe alguma alternativa melhor?